contracts/test/OracleTest.sol

```diff
@@ -0,0 +1,74 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity =0.7.6;
+
+import '../libraries/OracleLibrary.sol';
+
+contract OracleTest {
+    uint32[2] private POSITIVE_TICK_CASE = [uint32(11), uint32(0)];
```

Comment:
weird to hard code these test cases. ideally you write a separate contract that can be constructed to return values for any test case. then you dont touch the smart contracts while you work on unit tests